packages/framework/src/loader.ts

```diff
@@ -70,8 +73,20 @@ const getArtifacts = async (
         ...config,
         secretVariables: document.secrets as { [key: string]: unknown },
       };
+    } else if (document.type === "font") {
+      const font = document as { [key: string]: unknown };
```

Comment:
FontDTO type should reflect what the firebase document is like and should be what you cast to here.